bot/cogs/antispam.py

```diff
@@ -254,6 +263,27 @@ def mod_log(self) -> ModLog:
         await deletion_context.upload_messages(self.bot.user.id, self.mod_log)
 
 
+async def reupload_attachments(
+    bot: Bot,
+    message: Message,
+    channel_id: int = GuildConfig.attachment_repost
+) -> List[str]:
+    """Re-upload message's attachments to the the channel_id and return the list of re-posted attachments URLs."""
+    if not message.attachments:
+        return []
+    channel = bot.get_channel(channel_id)
+    out = []
+    for attachment in message.attachments:
+        try:
+            with BytesIO() as buffer:
+                await attachment.save(buffer, use_cached=True)
+                reupload: Message = await channel.send(file=File(buffer, filename=attachment.filename))
+                out.append(reupload.attachments[0].url)
+        except (HTTPException, NotFound):
+            log.warning(f"Tried to re-upload attchment {attachment.id}, but it has failed.")
```

Comment:
```suggestion
            log.warning(f"Tried to re-upload attachment {attachment.id}, but it has failed.")
```

Reply:
I think an attachment ID is meaningless information, at least on its own. It'd be helpful to include the message's ID too and/or the filename.